src/pages/api/events/[id]/register.js

```diff
@@ -0,0 +1,30 @@
+const { updateEventID } = require("../../../../../server/actions/events");
+const mailchimp = require("@mailchimp/mailchimp_transactional")(
+  process.env.MAILCHIMP_API_KEY
+);
+
+export default async function handler(req, res, next) {
+  if (req.method === "POST") {
+    const eventId = req.query.id;
+    const event = req.body.event;
+    const user = req.body.user;
+
+    await mailchimp.messages.sendTemplate({
+      template_name: "event-register-confirmation",
+      template_content: [],
+      message: {
+        to: [{ email: `${user.bio.email}` }],
+        global_merge_vars: [
+          {
+            name: "eventTitle",
+            content: `${event.title}`,
+          },
+        ],
+      },
+    });
+
+    const updatedEvent = await updateEventID(eventId, event);
+
```

Comment:
@jackpalaia  Here, should we first update the event and then send an email if update event is successful? If we send a confirmation email and then for some reason updateEventID fails, that would be an incorrect flow.